bool lock_position
bool lock_orientation
bool lock_grasp
float32[6] torque
float32[6] force
float32[2] grip_force